CONTRIBUTING.md

```
@@ -76,6 +49,7 @@ PRs that violate these safety principles will not be merged regardless of code q
 - [Before You Submit](#before-you-submit)
 - [Pull Request Process](#pull-request-process)
 - [Code Style](#code-style)
+- [Documentation](#documentation)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)